apps/dolly-frontend/src/main/js/src/components/fagsystem/pdlf/form/partials/innvandring/Innvandring.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+// @ts-ignore
+import { FormikSelect } from '~/components/ui/form/inputs/select/Select'
+// @ts-ignore
+import { AdresseKodeverk, PersoninformasjonKodeverk } from '~/config/kodeverk'
+import { FormikProps } from 'formik'
+import { FormikDollyFieldArray } from '~/components/ui/form/fieldArray/DollyFieldArray'
+import { AvansertForm } from '~/components/fagsystem/pdlf/form/partials/avansert/AvansertForm'
+import _get from 'lodash/get'
+import { FormikTextInput } from '~/components/ui/form/inputs/textInput/TextInput'
+import { FormikDatepicker } from '~/components/ui/form/inputs/datepicker/Datepicker'
+
+export interface InnvandringArray {
+	person: {
+		innvandretFraLand: Array<InnvandringValues>
+	}
+}
+
+interface InnvandringValues {
+	fraflyttingsland?: string
+	fraflyttingsstedIUtlandet?: string
+	innflyttingsdato?: Date
+}
+
+interface InnvandringProps {
+	formikBag: FormikProps<{ pdldata: InnvandringArray }>
+}
+
+const initialInnvandring = {
```

Comment:
Har en egen fil for pdlf initialValues, kan gjerne legge denne der, siden den blir brukt flere steder.